arrows/core/video_input_image_list.cxx

```diff
@@ -127,14 +136,25 @@ ::get_configuration() const
   vital::config_block_sptr config = vital::algo::video_input::get_configuration();
 
   config->set_value( "path", "",
-                     "Path to search for image file. "
-                     "If a file name is not absolute, this list of directories is scanned to find the file. "
-                     "The current directory '.' is automatically appended to the end of the path. "
-                     "The format of this path is the same as the standard "
-                     "path specification, a set of directories separated by a colon (':')" );
+    "Path to search for image file. "
+    "If a file name is not absolute, this list of directories is scanned to "
+    "find the file. The current directory '.' is automatically appended to "
+    "the end of the path. The format of this path is the same as the standard "
+    "path specification, a set of directories separated by a colon (':')" );
   config->set_value( "allowed_extensions", "",
-                     "Semicolon-separated list of allowed file extensions. "
-                     "Leave empty to allow all file extensions." );
+    "Semicolon-separated list of allowed file extensions. Leave empty to allow "
+    "all file extensions." );
+  config->set_value( "sort_by_time", "false",
+    "Instead of accepting the input list as-is, sort input file order based "
+    "on timestamp metadata contained either in the file or in each filename." );
+  config->set_value( "skip_bad_images", "false",
```

Comment:
Who uses this, and how? It seems like this will not help if reading the image metadata fails. I wonder if it doesn't make more sense for users to just catch the exception instead?

(Comments from other stakeholders would be appreciated!)

Reply:
2 NOAA people, when processing 100k images typically 3-4 are corrupted and throw errors within vil_load or other